omod/src/main/java/org/openmrs/module/appui/rest/SessionController.java

```diff
@@ -31,12 +33,10 @@
 
     @RequestMapping(method = RequestMethod.GET)
     @ResponseBody
-    public Object get(HttpServletRequest request, HttpServletResponse response) {
+    public Object get(HttpServletRequest request, HttpServletResponse response){
         RequestContext requestContext = RestUtil.getRequestContext(request, response);
         Representation rep = requestContext.getRepresentation();
 
-        locationService = Context.getLocationService();
-        providerService = Context.getProviderService();
```

Comment:
I thought that if locationService and providerService are Autowired we don't need those lines and it worked when I tested it